hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -554,6 +638,10 @@ public static OmBucketInfo getFromProtobuf(BucketInfo bucketInfo) {
         .setQuotaInBytes(bucketInfo.getQuotaInBytes())
         .setUsedNamespace(bucketInfo.getUsedNamespace())
         .setQuotaInNamespace(bucketInfo.getQuotaInNamespace());
+    if (bucketInfo.hasEcReplicationConfig()) {
+      obib.setEcReplicationConfig(
+          new ECReplicationConfig(bucketInfo.getEcReplicationConfig()));
+    }
     if (bucketInfo.hasObjectID()) {
       obib.setObjectID(bucketInfo.getObjectID());
     }
```

Comment:
There are a few other methods in OmBucketInfo I think need updated to account for the new ECReplicationType field. Eg:

`toString()`
`equals()` - seems to take all fields into account at the moment.
`getObjectInfo()`

Reply:
Done